dev-manuals/illustrator/Tile.md

```diff
@@ -0,0 +1,51 @@
+# How to make a Tile 
+
+## Setup
+
+### Step 1
+
+Open either the 32x32px template or the 16x16px template in Illustrator. 
+You can find the templates in the new-tiles repository in the templates folder.
+
+### Step 2
+
+Add the colorset you want.
+
+![](setup1.png)
+
+
+### Step 2
+
+Choose the right tool.
+
+![](setup2.png)
+
+
+### Step 3
+
+Choose the right color. left click to paint. have fun!
+And when drawing you have to make sure that the shadows are transparent.
+
+![](setup3.png)
+
+
+
+## Export 32x32px or 16x16px
+
+### Step 1
+
+![](32x32ex1.png)
+
+### Step 2
+
+![](32x32ex2.png)
+
+### Step 3
+
+![](32x32ex3.png)
+
+### Result
+
+![](32x32px_Zeichenfl%C3%A4che%201.png)
```

Comment:
I think just like the `setup*.png`s, these images should be renamed to `export*.png`.